packages/engine/Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -216,33 +216,46 @@ function ScreenSpaceCameraController(scene) {
     eventType: CameraEventType.LEFT_DRAG,
     modifier: KeyboardEventModifier.SHIFT,
   };
+
+  const ellipsoid = defaultValue(scene.ellipsoid, Ellipsoid.default);
+
   /**
    * The minimum height the camera must be before picking the terrain or scene content instead of the ellipsoid.
    * @type {number}
    * @default 150000.0
```

Comment:
All the docs for these properties indicate a default that's not technically true if the ellipsoid has been changed. Maybe they should be updated?